fingerprint/models.py

```diff
@@ -129,6 +140,8 @@ class RequestFingerprint(AbstractFingerprint):
     accept = models.CharField(max_length=255, blank=True)
     content_encoding = models.CharField(max_length=255, blank=True)
     content_language = models.CharField(max_length=255, blank=True)
+    referer = models.CharField(max_length=255, blank=True)
```

Comment:
btw, I only just realised, isn't referer like full blown URL? I wonder if it should allow for more than just 255 chars, maybe 2k? (it is still conservative limit for what is out there...)

Reply:
That's a good point. Increased to 2047

Reply:
Nah you can't do it like this!
https://stackoverflow.com/questions/2597964/what-is-the-max-size-of-max-length-in-django

Reply:
lies!

if I read this correctly neither postgresql nor mysql limits this to 255
I guess we should fix [django documentation next](https://docs.djangoproject.com/en/dev/ref/databases/#character-fields) :D